cmd/compare.go

```diff
@@ -18,6 +18,11 @@ import (
 )
 
 func Compare() {
+	passphrase := os.Getenv("PASSPHRASE")
```

Comment:
Les adjust this to be VAULTIFY_PASSPHRASE to prevent other systems that maye use PASSPHRASE or people may get confused what this environment variable is used for which tool on there system etc.